### PR TITLE
Updates and fixes

### PR DESCRIPTION
**Pull request by dennisarimi**
Updates from client feedback, general fixes and fixing team page width

### CODE REVIEW

**Comment by dennisarimi** [comment]
Thought; do we have the team page display a max of 3 team members horizontally? or should we leave it to fit as many as possible as it is in this pr?

**Comment by marksigilai** [comment]
for the team page pls try this css: might be all we need

.main {
	max-width: var(--max-width);
	width: 100%;
	min-height: 100vh;
}

.teamMembersContainer {
	display: flex;
	flex-wrap: wrap;
	justify-content: space-around;
	gap: 10px;
}

File: public/team_members.tsx [hunk 1]
```diff
@@ -4,219 +4,131 @@ export const teamMembers = [
         name: 'Prof. Isaac K. Nyamongo',
         position: 'Principal Investigator (PI)',
         description: 'Prof. Isaac K. Nyamongo is a distinguished Anthropology Professor and Deputy Vice-Chancellor at the Cooperative University of Kenya. With extensive research and training grants from prestigious organizations, including the EU and WHO, his work spans multiple African countries to focus on the social drivers of human development. Prof. Nyamongo is the current President of the World Anthropological Union.',
-        socialMedia: {
```
Comment (marksigilai) [inline public/team_members.tsx:7]: you'll need to remove the socialMedia prop from the team/page.tsx

Comment (dennisarimi) [reply public/team_members.tsx:7]: i did, you've commented on the old file😂😂

**Review by marksigilai — APPROVED** [review]

**Pull request merged**